publish.ps1

```diff
@@ -0,0 +1,44 @@
+[cmdletbinding()]
+Param(
+  [switch]$Publish
+)
+
+Begin {
+  $BuildFolder = "$PSScriptRoot\Puppet.Dsc"
+  $MarkdownDocsFolder = Resolve-Path "$PSScriptRoot\docs"
+
+  If (Test-Path -Path $BuildFolder) {
+    Remove-Item -Path $BuildFolder -Recurse -Force
+  }
+  New-Item -Path $BuildFolder -ItemType Directory | Out-Null
+}
+
+Process {
+  # Copy source files
+  $FoldersToCopy = @(
+    "$PSScriptRoot\src\en-us"
```

Comment:
Nit: Would like these to be join-path instead of interpolation, but could see the code to do that being too complicated to be easily read. In the future this will be fragile running in pwsh on other platfroms.

Reply:
Oh, fair!